src/ontology/imports/omo_import.owl

```diff
@@ -7,9 +7,9 @@ Prefix(rdfs:=<http://www.w3.org/2000/01/rdf-schema#>)
 
 
 Ontology(<http://purl.obolibrary.org/obo/mondo-ingest/imports/omo_import.owl>
-<http://purl.obolibrary.org/obo/mondo-ingest/releases/2023-11-30/imports/omo_import.owl>
+<http://purl.obolibrary.org/obo/mondo-ingest/releases/2024-01-11/imports/omo_import.owl>
```

Comment:
## `OMO` & `RO` (FYI)
<details><summary>Details</summary>
<p>

I needed to run the `make` pipeline to get the prereqs to run the lexmatch script. These files got updated.

I pushed to main already, and when I do the diff locally, it doesn't show any diff for these 2 files, so this feels like a GitHub bug to me.

</p>
</details>